### PR TITLE
Query and monitor if we have skipped any scheduled jobs

### PR DESCRIPTION
**Pull request by xiaohansong**
## What
https://github.com/airbytehq/airbyte-cloud/issues/2225

Monitor if we have skipped any scheduled jobs

### CODE REVIEW

**Review by git-phu — APPROVED** [review]

File: airbyte-metrics/metrics-lib/src/main/java/io/airbyte/metrics/lib/MetricQueries.java [hunk 1]
```diff
@@ -148,4 +148,44 @@ public static List<Pair<JobStatus, Double>> overallJobRuntimeForTerminalJobsInLa
     return pairedRes;
   }
 
+  public static Long numOfJobsNotRunningOnSchedule(final DSLContext ctx) {
```
[the remark this answers is not in the document]
Comment (xiaohansong) [reply airbyte-metrics/metrics-lib/src/main/java/io/airbyte/metrics/lib/MetricQueries.java:151]: not in unit test I'm afraid... but I ran these manually on dbeaver.

[the remark this answers is not in the document]
Comment (git-phu) [reply airbyte-metrics/metrics-lib/src/main/java/io/airbyte/metrics/lib/MetricQueries.java:151]: ok, should we add a comment about what our logic is for determining if a job run was missed?

[the remark this answers is not in the document]
Comment (xiaohansong) [reply airbyte-metrics/metrics-lib/src/main/java/io/airbyte/metrics/lib/MetricQueries.java:151]: make sense. I have a runbook coming in https://github.com/airbytehq/airbyte-cloud/pull/2272; I'll add a short summary here.

[the remark this answers is not in the document]
Comment (git-phu) [reply airbyte-metrics/metrics-lib/src/main/java/io/airbyte/metrics/lib/MetricQueries.java:151]: Oh ok perfect, thanks!

**Pull request merged**